src/13_file_io.py

```diff
@@ -10,10 +10,14 @@
 # Note: pay close attention to your current directory when trying to open "foo.txt"
 
 # YOUR CODE HERE
+with open('foo.txt', 'r') as txt:
+    print(txt.readlines())
```

Comment:
Good. 

If you haven't already, I want you to try to use a for-loop here to access and print each line in the file. This will come up on future assignments/sprint challenges.